sagemaker/run/train_entry.py

```diff
@@ -37,10 +37,11 @@ def get_train_parser():
         required=True)
     parser.add_argument("--train-yaml-s3", type=str,
         help="S3 location of training yaml file. "
-             "Do not store it with partitioned graph",
-             required=True)
-    parser.add_argument("--model-artifact-s3", type=str,
-        help="S3 location to store the model artifacts.")
+             "Do not store it with partitioned graph")
+    parser.add_argument("--model-artifact-s3", type=str, default=None,
+        help="S3 location to store the model artifacts. If None, we rely on SageMaker "
+        "to upload model artifacts, so the launching Estimator needs to have 'output_path' set. "
+        "Default: None")
```

Comment:
Maybe we should set it to `required=True`.

Reply:
If we do that we won't be able to set the value to `None`, but we could change it to require a specific string. Not sure which is better UX.